recipes/py2dm/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "py2dm" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/py2dm-{{ version }}.tar.gz
+  sha256: 447e3af1e5adfc7a898acd5e5744ab58f86d995cf615147629fec7b4bf605be7
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 1
+
+requirements:
+  build:
+    - typing-extensions   # [py<38]
+    - cached-property     # [py<38]
+    - setuptools
+  host:
+    - python
+    - typing-extensions   # [py<38]
+    - cached-property     # [py<38]
+    - pip
+  run:
+    - python
+    - typing-extensions   # [py<38]
+    - cached-property     # [py<38]
+    - libcxx              # [osx]
+
+test:
+  imports:
+    - py2dm
```

Comment:
It looks like this provides both a Python and compiled implementation. I presume you want the compiled one to work in conda-forge?

If so, is there a way of testing to make sure it was compiled rather than it silently falling back (like the Linux build currently is).

Reply:
Great idea, I'm writing a more thorough test case and will commit your other suggestions.